product-service/src/functions/getProductsById/handler.ts

```diff
@@ -1,18 +1,26 @@
-import type { ValidatedEventAPIGatewayProxyEvent } from '../../libs/api-gateway';
-import { formatJSONResponse, errorNotFoundResponse, errorServiceUnavailableResponse } from '../../libs/api-gateway';
-import { middyfy } from '../../libs/lambda';
-import mockData from '../mock.json';
+import type {ValidatedEventAPIGatewayProxyEvent} from '../../libs/api-gateway';
+import {formatJSONResponse, errorNotFoundResponse, errorServiceUnavailableResponse} from '../../libs/api-gateway';
+import {middyfy} from '../../libs/lambda';
 import productSchema from './schema';
+import {Client} from 'pg';
+import dbOptions from '@libs/db-options';
 
 const getProductsById: ValidatedEventAPIGatewayProxyEvent<typeof productSchema> = async (event) => {
+  const client = new Client(dbOptions);
   let product;
 
+  console.log(event);
+
+  await client.connect();
+
   try {
-    const products = await Promise.resolve(mockData.body);
+    const data = await client.query('select * from products inner join stocks on id = product_id');
```

Comment:
It is better to use outer join to see the products without records in stocks table